capella2polarion/__main__.py

```diff
@@ -81,13 +81,20 @@ def print_cli_state(capella2polarion_cli: Capella2PolarionCli) -> None:
 @click.option("--force-update", is_flag=True, default=False)
 @click.option("--type-prefix", type=str, default="")
 @click.option("--role-prefix", type=str, default="")
+@click.option(
+    "--grouped-links-custom-fields",
+    envvar="CAPELLA2POLARION_GROUPED_LINKS_CUSTOM_FIELDS",
+    is_flag=True,
+    default=False,
+)
```

Comment:
This should default to True, so that by default it behaves just like before this PR.

```suggestion
@click.option(
    "--grouped-links-custom-fields / --no-grouped-links-custom-fields",
    envvar="CAPELLA2POLARION_GROUPED_LINKS_CUSTOM_FIELDS",
    is_flag=True,
    default=True,
)
```